src/app/wizard/step/provider-settings/provider/extended/kubevirt/template.html

```diff
@@ -97,15 +101,17 @@
                 <mat-error *ngIf="control.get(Controls.PreAllocatedDataVolumeUrl).hasError( 'required')">
                   <strong>Required</strong>
                 </mat-error>
+                <mat-error *ngIf="control.get(Controls.PreAllocatedDataVolumeUrl).hasError( 'pattern')">
```

Comment:
**Cutting below text**

![overlapping](https://user-images.githubusercontent.com/17727069/199423000-ddc31c30-59e1-4663-88ad-f777766679ec.png)


Is this valid Value for **URL**?
![valid-url](https://user-images.githubusercontent.com/17727069/199423069-3f06f6ef-21d7-45f0-a7e6-c2832733ca43.png)

Reply:
Opearting system URL should not be validated against lowercase RFC 1123

Reply:
if you want to check URL you can use js URL constructor